### DISCUSSION TITLE
How many preload.js file is configured in Electron

### DISCUSSION THREAD

**Discussion by azhengyongqin** [quasarframework/quasar#12836] [CLI - Electron mode]
I'm using [Quasar CLI(with Vite)](https://quasar.dev/quasar-cli-vite/developing-electron-apps/configuring-electron) Developing Electron APP.

My APP needs to use the `Electron` `BrowserWindow` and `BrowserView` components, which need to load different `preload.js` files respectively.

But I don't know how to configure multiple `preload.js` files in `quasar.config.js`, I just get such configuration in the documentation, but it could not meet my needs.

```js
sourceFiles: {
  electronMain: 'src-electron/electron-main',
  electronPreload: 'src-electron/electron-preload'
}
```

I need to have such a configuration：

```js
sourceFiles: {
  electronMain: 'src-electron/electron-main',
  electronPreload: [
    'preload1':'src-electron/electron-preload1',
    'preload2':'src-electron/electron-preload2',
  ]
}
```

**Comment by heyauri** [comment]
Hey, have you addressed this problem yet? I find i have the same demand.

**Comment by wb-wenbei** [comment]
+1
